GitUI/UserControls/FileStatusList.Designer.cs

```diff
@@ -57,18 +58,18 @@ private void InitializeComponent()
             this.FileStatusListView.ShowItemToolTips = true;
             this.FileStatusListView.Size = new System.Drawing.Size(682, 464);
             this.FileStatusListView.Sorting = System.Windows.Forms.SortOrder.Ascending;
-            this.FileStatusListView.TabIndex = 2;
+            this.FileStatusListView.TabIndex = 4;
```

Comment:
I see the tab order has changed.
I just want a confirmation the tab order remains logical

Reply:
This is an intended change. `(Shift+)Tab` works correctly.

Reply:
Thank you for confirming